example/src/index.ts

```diff
@@ -54,14 +55,7 @@ function main(): void {
   dock.spacing = 8;
 
   fbWidget.openRequested.connect((fb, path) => {
```

Comment:
@jasongrout, the notebook opener can just hook up to `openRequested` and use that, we can just filter on path.